test/Vault.spec.ts

```diff
@@ -1272,6 +1316,30 @@ describe('Vault', () => {
   });
 
   describe('deposit', () => {
+    describe('using an underlying that charges fees on transfer like USDC', () => {
```

Comment:
add "reverts" or "fails"

Reply:
nope, that's on the `it` part, this is just context

Reply:
oh sry, my mistake